ampersand-select-view.js

```diff
@@ -173,8 +182,20 @@ SelectView.prototype.remove = function () {
     this.el.removeEventListener('change', this.onChange, false);
 };
 
+SelectView.prototype.reset = function() {
+    if(this.startingValue) {
+        this.setValue(this.startingValue, true);
+    } else {
+        this.clear();
+    }
+};
+
 SelectView.prototype.clear = function() {
-    this.setValue('', true);
+    if(this.unselectedText) {
+        this.setValue(undefined, true);
```

Comment:
hey @nickryall, that's up for discussion!  I think it makes sense to be consistent with [DOM select values](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElementhttps://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement) and yield a consistent returned type out of the interface.  Because `''` is falsy, I think it simplifies the module.  If we had to return anything else, maybe not so much.